tests/main/views/test_buyer_domains.py

```diff
@@ -158,6 +158,13 @@ def test_create_buyer_email_domain_fails_if_higher_level_domain_already_approved
 
 
 class TestListBuyerEmailDomains(BaseApplicationTest):
+    def setup(self):
+        super(TestListBuyerEmailDomains, self).setup()
+        with self.app.app_context():
+            # Remove default fixture so we can test pagination
```

Comment:
Looking at the below tests, all of them require there to be no domains in the table, so the comment should be more general about wanting to start from an empty state. With that being the case, is it worth deleting everything in the table rather than specifically mentioning the one domain which we assume will be there (and be the only domain there)?

Reply:
Yep, good call. Will fix